src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -80,6 +81,20 @@ object Boot extends IOApp with LazyLogging {
 
         _ <- dependencies.policyEvaluatorService.initPolicy()
 
+        // make sure all users referenced by resourceAccessPolicies exist
+        _ <- appConfig.resourceAccessPolicies.flatMap { case (_, policy) => policy.memberEmails }.toList.traverse { email =>
+          dependencies.samApplication.userService.inviteUser(email, SamRequestContext()).attempt
+        }
+
+        // create resourceAccessPolicies but don't prevent startup if any fail
+        policyTrials <- dependencies.samApplication.resourceService.upsertResourceAccessPolicies(appConfig.resourceAccessPolicies)
+        _ = policyTrials.map {
+          case Left(t) =>
+            logger.error("NON-FATAL - failure creating configured policy on startup, continuing", t)
```

Comment:
It seemed drastic to have sam not start up because of this. I suppose with k8s there would not be down time, but it would still be disruptive to have to deal with it. I was not thinking about it as Sam not behaving as expected. I was thinking that Sam does not really care what the policies are. Your call, I am happy to go either way and your team supports it.

Reply:
I think I'd rather have it be disruptive. A bee would not start up if a policy can't be upserted, and we require a successful `create-bee` action to succeed before merging. I agree Sam shouldn't care what the policies are, but if a policy defined in `reference.conf` can't be upserted, I would consider that an error state.